modules/redpanda/redpanda_test.go

```diff
@@ -84,6 +84,119 @@ func TestRedpandaWithAuthentication(t *testing.T) {
 	container, err := RunContainer(ctx,
 		WithEnableSASL(),
 		WithEnableKafkaAuthorization(),
+		WithEnableWasmTransform(),
+		WithNewServiceAccount("superuser-1", "test"),
+		WithNewServiceAccount("superuser-2", "test"),
+		WithNewServiceAccount("no-superuser", "test"),
+		WithSuperusers("superuser-1", "superuser-2"),
+		WithEnableSchemaRegistryHTTPBasicAuth(),
+	)
+	require.NoError(t, err)
+	// }
+
+	// Clean up the container after the test is complete
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	// kafkaSeedBroker {
+	seedBroker, err := container.KafkaSeedBroker(ctx)
+	// }
+	require.NoError(t, err)
+
+	// Test successful authentication & authorization with all created superusers
```

Comment:
Could you break up these tests into different test table entries or test methods, whatever you prefer? That would add readability on failures 🙏

Reply:
The only change I made with testing in this file was to enable the flag on an existing test, then duplicate and rename that existing test so that I could change the versioning on it. 

I did just now add a simple test of the new versioning function so that is in there and following the table based testing methodology. But if possible I'd like to avoid a major rewrite of a test file I don't have much familiarity with to make a fairly minor change to this codebase.

Reply:
I'd like to have test tables with a for loop with `t.Run` instead of code blocks for each case 🙏

Reply:
oh wait wait, let me step back. I double checked that the code is already in that format, so please forgive me. I think this is fine, and we can do a refactor in a follow-up PR.